### PR TITLE
Make CodeFormatter work with multiline comments

### PR DESCRIPTION
**Pull request by ptomin**

### CODE REVIEW

File: src/Core/StringEx.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Reko.Core
+{
```
Comment (uxmal) [inline src/Core/StringEx.cs]: Consider inlining this in the class that is using it. Only if we find that we are splitting into lines in many places does it make sense to create an extension class for `System.String`.

Comment (ptomin) [reply src/Core/StringEx.cs]: I'm going to use it at `CodeFormatter` also

Comment (uxmal) [reply src/Core/StringEx.cs]: I'm still unhappy about tacking this onto the `string` class. Consider using it inlining it in the two places you're using it.

Comment (ptomin) [reply src/Core/StringEx.cs]: OK

**Comment by uxmal** [comment]
One unit test failing currently.

**Comment by ptomin** [comment]
It's OK. I have created it to reproduce error

**Comment by ptomin** [comment]
Ready to merge?

**Pull request merged**